app/src/main/java/org/stacks/app/data/EncryptedPreferencesIdentityRepository.kt

```diff
@@ -54,9 +56,10 @@ class EncryptedPreferencesIdentityRepository
         try {
             val identities = JSONArray(jsonArrayString)
 
-            for (i in 0 until identities.length()) {
-                listOfIdentities.add(IdentityModel(identities.getJSONObject(i)))
+            identities.forEach<JSONObject> {
+                listOfIdentities.add(IdentityModel(it))
             }
```

Comment:
A `map` would be simper than a `forEach`.

Reply:
doesn't map seems like the initial object is changed my the given function? or at least returns a different object?